Fxi broken links at design/baremetal-operator/annotation-for-power-cycling-and-deleting-failed-nodes.md

```
Errors in design/baremetal-operator/annotation-for-power-cycling-and-deleting-failed-nodes.md
[ERROR] file:///home/runner/work/metal3-docs/metal3-docs/design/baremetal-operator/github.com/kubevirt/machine-remediation | Cannot find file
[ERROR] file:///home/runner/work/metal3-docs/metal3-docs/design/baremetal-operator/github.com/metal3-io/cluster-api-provider-baremetal | Cannot find file
```

Probably is missing `./` to indicate local link.

/triage accepted
/help
/good-first-issue

--- a/design/baremetal-operator/annotation-for-power-cycling-and-deleting-failed-nodes.md
+++ b/design/baremetal-operator/annotation-for-power-cycling-and-deleting-failed-nodes.md
@@ -118,8 +118,8 @@ See [PoC code](https://github.com/kubevirt/machine-remediation/)
 ### Work Items
 
 - Make any requested modifications
-- Create a PR from [KubeVirt](github.com/kubevirt/machine-remediation)
-  into [CAPM3](github.com/metal3-io/cluster-api-provider-baremetal)
+- Create a PR from [KubeVirt](https://github.com/kubevirt/machine-remediation)
+  into [CAPM3](https://github.com/metal3-io/cluster-api-provider-baremetal)
 
 ### Dependencies
 

